### PR TITLE
DOCS-2079

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

DOCS-2079

### Motivation

N/A

### Preview

https://docs-staging.datadoghq.com/alai97/rum-troubleshooting/real_user_monitoring/browser/troubleshooting

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/browser/troubleshooting.md [hunk 1]
```diff
@@ -10,16 +10,17 @@ further_reading:
...
+| Ad blockers prevent the Browser RUM SDK from being downloaded or sending data to Datadog.     | Some ad blockers extend their restrictions to performance and marketing tracking tools. See the [Install the Browser RUM SDK with npm][3] and [forward the collected data through a proxy][4] docs. |
+| Network rules or VPNs prevent the Browser RUM SDK from being downloaded or sending data to Datadog. | Grant access to the endpoints required to download the SDK or to send data. The list of endpoints is available in the [Content Security Policy documentation][5].                                        |
+| Scripts, packages, and clients initialized before the Browser RUM SDK can lead to missed logs, resources, and user actions. | Check when/where the Browser RUM SDK is initialized and consider moving this step earlier in the execution of your application code.                                             |
```
Comment (sarina-dd) [inline content/en/real_user_monitoring/browser/troubleshooting.md:23]:
```suggestion
| Scripts, packages, and clients initialized before the Browser RUM SDK can lead to missed logs, resources, and user actions. | Check when or where the Browser RUM SDK is initialized and consider moving this step earlier in the execution of your application code.                                             |   
```

Comment (sarina-dd) [reply content/en/real_user_monitoring/browser/troubleshooting.md:23]: Small removal for translation

File: content/en/real_user_monitoring/browser/troubleshooting.md [hunk 2]
```diff
@@ -10,16 +10,17 @@ further_reading:
       text: 'Content Security Policy'
 ---
 
-If you experience unexpected behavior with Datadog Browser RUM, this guide may help resolve issues quickly. Reach out to [Datadog support][1] for further assistance. Regularly update to the latest version of the [RUM Browser SDK][2], as each release contains improvements and fixes.
+If you experience unexpected behavior with Datadog Browser RUM, use this guide to resolve issues quickly. If you continue to have trouble, contact [Datadog support][1] for further assistance. Datadog recommends regularly updating to the latest version of the [RUM Browser SDK][2], as each release contains improvements and fixes.
```
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/troubleshooting.md:13]:
```suggestion
If you experience unexpected behavior with Datadog Browser RUM, use this guide to resolve issues quickly. If you continue to have trouble, contact [Datadog support][1] for further assistance. Regularly update to the latest version of the [RUM Browser SDK][2], as each release contains improvements and fixes.
```

The other changes are great. Avoid if at all possible when you can write it some other way: a) referring to the company by name in the docs, and b) "recommending"  things (yes, the word appears a lot in our docs, but it usually doesn't need to -- certainly don't introduce it).

File: content/en/real_user_monitoring/browser/troubleshooting.md (hunk 1 above)
Comment (kayayarai) [inline content/en/real_user_monitoring/browser/troubleshooting.md:23]:
```suggestion
| Scripts, packages, and clients initialized before the Browser RUM SDK can lead to missed logs, resources, and user actions. | Check where in the code the Browser RUM SDK is initialized and consider moving this step earlier in the execution of your application code.  For example, if the Browser RUM SDK is not initialized before ApolloClient initialization, `graphql` requests (shown as XHR resources in RUM Explorer), might not be logged.                                           |   
```
I think you should keep the concrete example provided in the Jira card; it makes both the problem and the fix clearer.

**Review by kayayarai — CHANGES_REQUESTED** [review]
Comments left!

**Review by kayayarai — APPROVED** [review]

**Pull request merged**